sdk/storage/storage-blob/test/containerurl.spec.ts

```diff
@@ -234,7 +240,10 @@ describe("ContainerURL", () => {
   it("listBlobHierarchySegment with default parameters", async () => {
     const blobURLs = [];
     for (let i = 0; i < 3; i++) {
-      const blobURL = BlobURL.fromContainerURL(containerURL, recorder.getUniqueName(`blockblob${i}/${i}`));
+      const blobURL = BlobURL.fromContainerURL(
+        containerURL,
```

Comment:
Auto-formatted due to VSCode

Reply:
Are you not using Prettier and ESLint extensions for VSCode ?

Reply:
I am using it indeed! What's the issue?

Reply:
Fine. 
Pulled your code, these changes do not create any formatting conflicts in VS Code for me.

Reply:
In .vscode/settings=>    you can disable the
"editor.formatOnSave": true, 
    set it to false